src/docker/Dockerfile

```diff
@@ -32,6 +32,7 @@ LABEL Description="This is the image for aether.jcabi.com" Vendor="jcabi.com" Ve
 WORKDIR /tmp
 
 ENV DEBIAN_FRONTEND=noninteractive
+ENV OPENJDK_VERSION 1.7.0_75
```

Comment:
@mkordas Actually, it's not - I tried building the image locally as u suggested and it cannot find the version ``1.7.0_75`` . I will look more into it today, to find a propper version. It sucks because Java versioning system was never quite clear to me... different names/numbers for the same thing etc etc. If you can suggest an openjdk7 version it would be cool :D